src/events/crawler/scrapers/AUS/index.js

```diff
@@ -5,8 +5,21 @@ import * as transform from '../../lib/transform.js';
 // Set county to this if you only have state data, but this isn't the entire state
 // const UNASSIGNED = '(unassigned)';
 
+const idMap = {
+  'Australian Capital Territory': 'id3:AU-ACT',
```

Comment:
Not sure what these 0 and 3s are doing. Why not build on AUS and ACT as a concatenation for example? I think I’m not quite following the intent here, most likely my fault but would love to clarify.

Reply:
It's just 100% mirroring the country, unit, subunit and state levels in Natural Earth dataset. Basically Natural Earth is the standard public source dataset for country shapes and it has this hierarchy.
If you look at https://github.com/hyperknot/country-levels/blob/master/docs/country_list.md it's basically Natural Earth's hierarchy mapped to id0, id1, id2 and id3.